apps/web/src/app/[orgShortcode]/convo/_components/convo-list-item.tsx

```diff
@@ -181,7 +181,7 @@ export const ConvoItem = memo(function ConvoItem({
               </div>
 
               <span className="line-clamp-2 overflow-ellipsis whitespace-break-spaces break-words">
-                {convo.entries[0]?.bodyPlainText ?? ''}
+                {convo.entries[0]?.bodyPlainText.trimStart() ?? ''}
```

Comment:
```suggestion
                {(convo.entries[0]?.bodyPlainText ?? '').trimStart()}
```
trimStart might throw if there is no body

Reply:
I have pushed a change a few hours ago to use trim() alone.

Reply:
use it like this `(convo.entries[0]?.bodyPlainText ?? '').trim()`